source/connect/general-connection.txt

```diff
@@ -11,3 +11,80 @@ General Connection
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`General` connection tab allows you to select a 
+:guilabel:`Connection String Scheme` for your |compass-short| deployment. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`General` tab.
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Select a Connection String Scheme.
+
+            You can select one of the following options:
+
+            .. list-table::
+               :header-rows: 1
+               :class: border-table
+
+               * - Connection String Scheme
+                 - Definition
+
+               * - mongodb
+                 - :manual:`Standard Connection String Format </reference/connection-string/>`. 
+                   The standard format of the MongoDB connection URI is used to 
+                   connect to a MongoDB deployment: standalone, replica set, or a sharded 
+                   cluster. 
+
+               * - mongodb+srv 
+                 - :manual:`DNS Seed List Connection Format 
+                   </reference/connection-string/#dns-seed-list-connection-format>`. 
+                   The ``+srv`` indicates to |compass| that the hostname that follows 
+                   corresponds to a DNS SRV record.
+
+         .. step:: Enter a Hostname.
+
+            In the :guilabel:`Host` field, enter the hostname of the server where the 
+            deployment is running.
+
+            If you are running your deployment locally, this value is ``localhost``. 
+            If you are connecting to an Atlas cluster, see 
+            :atlas:`Connect to Database Deployment </connect-to-database-deployment/>` 
+            for Atlas-specific guidance on how to connect to |compass-short|.
+
+            If you are not sure of your hostname, contact your Database 
+            Administrator for information.
+
+         .. step:: Enable Direct Connection. 
```

Comment:
Should this heading also include some indication that it is optional, like the advanced connection options step below?

Reply:
Yes, I agree!